src/abstractions/doppler-legacy-client/index.ts

```diff
@@ -10,6 +10,18 @@ export type UploadImageError =
   | { reason: "unexpected"; details: unknown };
 export type UploadImageResult = Result<void, UploadImageError>;
 
+export type PromoCodeItem = {
+  code: string;
+  type: "percent" | "money";
+  value: number;
+  startDate: Date | undefined;
+  endDate: Date | undefined;
+  useLimit: number | undefined;
+  minPaymentAmount: number;
+  promotionName: string;
+  isActive: boolean;
```

Comment:
These fields are not being considered from [unlayer-extension side](https://github.com/FromDoppler/unlayer-editor/blob/main/src/customJs/properties/promo_codes/types.ts#L10), but I think that they are not necessary, right?

Reply:
You are right, at the moment they are not necessary but I think that they could be useful in the future.